### PR TITLE
permissions object added to a repository

### PR DESCRIPTION
**Pull request by allvo**
This nested permissions object is available in REST API response: https://developer.github.com/v3/repos/ for a repository. Just reflecting this in the library code.

### CODE REVIEW

File: org.eclipse.egit.github.core/src/org/eclipse/egit/github/core/Permissions.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package jetbrains.charisma.persistent.globalSettings.org.eclipse.egit.github.core;
```
Comment (VladRassokhin) [inline org.eclipse.egit.github.core/src/org/eclipse/egit/github/core/Permissions.java]: Package seems incorrect.

Comment (allvo) [inline org.eclipse.egit.github.core/src/org/eclipse/egit/github/core/Permissions.java]: corrected, thanks for noticing.

**Comment by patoupatou** [comment]
Was this properly pushed to gerrit? As you may know, Egit doesn't accept PRs from Github, it's merely a mirror from the Foundation's gerrit (which you should be using if submitting patches).

**Comment by allvo** [comment]
@patoupatou Thanks for the info, frankly I did not know about it.

**Pull request closed without merging**